src/components/tabs/style.scss

```diff
@@ -12,13 +19,24 @@
 	}
 
 	&__heading--active {
-		color: #000;
-		border-bottom: 3px solid #377eb6;
+		.theme-light & {
+			color: $blue-wordpress-700;
+			border-bottom: 1px solid $light-gray-100;
+		}
+		.theme-dark & {
+			color: $blue-medium-200;
+			border-bottom: 1px solid $dark-gray-400;
```

Comment:
This border doesn't seem to be visible, maybe the colors are around the wrong way.